src/pages/services.jsx

```diff
@@ -36,7 +39,29 @@ export default function Services() {
 			<StyledWrapper>
 				<StyledUl>
 					{services.map(service => {
-						return <StyledLi key={service.id}>{service.serviceName}</StyledLi>;
+						return (
+							<>
+								<StyledLi key={service.id}>
+									<StyledP>
+										<StyledSpan>Service Name :</StyledSpan>
+										{service.serviceName}
+									</StyledP>
+									<StyledP>
+										<StyledSpan>Service Description : </StyledSpan>
+										{service.serviceDescription}
+									</StyledP>
+									<StyledP>
+										<StyledSpan>Service Price : </StyledSpan>
+										{service.servicePricePerHour}
+									</StyledP>
+									<StyledP>
+										<StyledSpan>Service Tax Key :</StyledSpan>
+										{service.serviceTaxKey}
+									</StyledP>
+									<StyledEditButton>Edit</StyledEditButton>
+								</StyledLi>
+							</>
```

Comment:
You don't need the fragments here and I think they might add some problems 🤔. As you would need to give the fragment the `key`. But since you only have one return value (`StyledLi`) you don't need the fragments anyways.
```suggestion
								<StyledLi key={service.id}>
									<StyledP>
										<StyledSpan>Service Name :</StyledSpan>
										{service.serviceName}
									</StyledP>
									<StyledP>
										<StyledSpan>Service Description : </StyledSpan>
										{service.serviceDescription}
									</StyledP>
									<StyledP>
										<StyledSpan>Service Price : </StyledSpan>
										{service.servicePricePerHour}
									</StyledP>
									<StyledP>
										<StyledSpan>Service Tax Key :</StyledSpan>
										{service.serviceTaxKey}
									</StyledP>
									<StyledEditButton>Edit</StyledEditButton>
								</StyledLi>
```